manim/cli/render/commands.py

```diff
@@ -137,10 +137,36 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
-    if config.notify_outdated_version or args["notify_outdated_version"]:
-        manim = requests.get("https://pypi.org/pypi/manim/json")
-        releases = manim.json()["releases"].keys()
-        stable = max(releases)
+    if config.notify_outdated_version:
+        manim_info_url = "https://pypi.org/pypi/manim/json"
+        warn_prompt = "Cannot check if latest release of manim is installed"
+        req_info = {}
+        releases = {}
+        stable = ""
+
+        try:
+            req_info = requests.get(manim_info_url, timeout=3)
+            req_info.raise_for_status()
+        except requests.exceptions.HTTPError:
+            logger.warning("HTTP Error: %s", warn_prompt)
+            raise SystemExit
+        except requests.exceptions.ConnectionError:
+            logger.warning("Connection Error: %s", warn_prompt)
+            raise SystemExit
+        except requests.exceptions.Timeout:
+            logger.warning("Timeout Error: %s", warn_prompt)
+            raise SystemExit
+        except Exception:
+            logger.warning("Something else went wrong: %s".warn_prompt)
```

Comment:
This one has a `.` instead of a `,`
```suggestion
            logger.warning("Something else went wrong: %s",warn_prompt)
```
But generally I would prefer using f-strings here (non-blocking):
```suggestion
            logger.warning(f"Something else went wrong: {warn_prompt}")
```

Reply:
This doesn't seem to be resolved yet.

Reply:
Just pushed 👍🏻

Reply:
> 
> 
> This doesn't seem to be resolved yet.

I only cared about the actual error with `,` vs `.`, but f-strings are appreciated!